be/src/exec/json_scanner.h

```diff
@@ -178,7 +178,14 @@ class JsonReader {
     std::vector<std::vector<JsonPath>> _parsed_jsonpaths;
     std::vector<JsonPath> _parsed_json_root;
 
-    rapidjson::Document _origin_json_doc; // origin json document object from parsed json string
+    char _value_buffer[4 * 1024 * 1024];
```

Comment:
Does it seem that this class may/should/must not be declared as a stack variable?

Reply:
It is not used on stack except ut.